packaging/opensuse-42.2/permissions.paranoid

```diff
@@ -0,0 +1 @@
+/usr/lib/snapd/snap-confine                             root:root         755
```

Comment:
No, paranoid permissions must strip all setuid bits even at the cost of breaking functionality.